node/src/service.rs

```diff
@@ -69,15 +71,15 @@ pub trait IdentifyVariant {
 
 impl IdentifyVariant for Box<dyn ChainSpec> {
 	fn is_kusama(&self) -> bool {
-		self.id().starts_with("kusama")
+		self.id().contains("kusama")
 	}
 
 	fn is_polkadot(&self) -> bool {
-		self.id().starts_with("polkadot")
+		self.id().contains("polkadot")
 	}
 
 	fn is_dev(&self) -> bool {
-		self.id().starts_with("dev")
+		self.id().contains("dev")
```

Comment:
this will match 'pint-dev' as 'dev' as well and blocking 'polkadot-launch' and e2e tests connecting to the relay chain, see the finalization failed in CI

Reply:
I see, so we should check if this is equal to `pint_dev` instead?